backends/bmv2/bmv2stf.py

```diff
@@ -613,7 +613,7 @@ def openInterface(ifname):
                 for interface, fp in self.interfaces.iteritems():
                     fp.close()
                 # Give time to the model to execute
-                time.sleep(2)
+                time.sleep(1)
```

Comment:
This probably doesn't belong in this change -- this sleep is an inherent race condition in running bmv2 (we have no "wait until done" command), and depends on how loaded the test machine is.  IIRC we changed this from 1 second to 2 originally as occasionally Travis jobs would fail with the 1 second delay when the Travis machines were heavily loaded.